frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/data-volume-wrapper.ts

```diff
@@ -106,6 +106,18 @@ export class DataVolumeWrapper extends K8sResourceObjectWithTypePropertyWrapper<
     return this;
   };
 
+  setPesistentVolumeClaimName = (name: string) => {
```

Comment:
What if `name` is null or undefined ?
Is there any need to validate the name (to check for non-allowed chars) ?

Reply:
we don't do this checks in other places ... IMHO it's ok to let the user of this method do the checks before calling this method